expo/gbe/templates/gbe/act_bid_review_list.tmpl

```diff
@@ -0,0 +1,123 @@
+{% extends 'base.tmpl' %}
+{% load staticfiles %}
+
+{% block title %}
+   Review Bids
+{% endblock %}
+
+{% block content %}
+
+
+  <p class="vanilla">{{title}} </p>
+  <h2 class="review-title">Bid Information for {{conference}}</h2>
+  <br/>
+      {% include "conference_picker.tmpl" %}
+  <br/>
+  <div>Press the arrows to sort on a column.  To sort on an additional sub-order,
+  press "Shift" to sort a secondary column.</div>
+  <br/>
+  <div>
+    Toggle column(s):
+    {% for header_item in header %}
+      <a class="toggle-vis" data-column="{{ forloop.counter0 }}">
+	{{ header_item }}</a>{% if not forloop.last %} - {% endif %}
+    {% endfor %}
+  </div>
+  <br/>
+  <table id="bid_review" class="order-column" cellspacing="0" width="100%">
+    <thead>
+      <tr class="bid-table">
+      {% for header_item in header %}
+        <th class="bid-table">{{ header_item }}</th>
+      {% endfor %}
+      </tr>
+    </thead>
+    <tfoot>
+      <tr class="bid-table">
+      {% for header_item in header %}
+        <th class="bid-table">{{ header_item }}</th>
+      {% endfor %}
+      </tr>
+    </tfoot>
+    <tbody>
+    {% for row in rows %}
+      <tr class="bid-table">
+              {% for value in row.bid %}
+	        <td class="bid-table">
+                  {{ value|linebreaks }}
+          	</td>
+	      {% endfor %}
+          {% if "Reviews" in header %}
+          <td class="bid-table">
+ 	  {% for review in row.reviews %}
+               {% if not forloop.first %}<hr/>{% endif %}
+                <b>Reviewer:</b> {{ review.evaluator.display_name }}<br>
+                <b>First Recommendation:</b> {{ review.primary_vote.show }}:
+		{{review.primary_vote.get_vote_display }}<br>
+                 <b>Second Recommendation:</b> {{ review.primary_vote.show }}:
```

Comment:
Good catch, will fix


On Wed, Sep 7, 2016 at 9:47 PM, bethlakshmi <notifications@github.com>
wrote:

> In expo/gbe/templates/gbe/act_bid_review_list.tmpl
> <https://github.com/bethlakshmi/GBE2/pull/883#discussion_r77934812>:
>
> > +    <tbody>
> > +    {% for row in rows %}
> > +      <tr class="bid-table">
> > +              {% for value in row.bid %}
> > +	        <td class="bid-table">
> > +                  {{ value|linebreaks }}
> > +          	</td>
> > +	      {% endfor %}
> > +          {% if "Reviews" in header %}
> > +          <td class="bid-table">
> > + 	  {% for review in row.reviews %}
> > +               {% if not forloop.first %}<hr/>{% endif %}
> > +                <b>Reviewer:</b> {{ review.evaluator.display_name }}<br>
> > +                <b>First Recommendation:</b> {{ review.primary_vote.show }}:
> > +		{{review.primary_vote.get_vote_display }}<br>
> > +                 <b>Second Recommendation:</b> {{ review.primary_vote.show }}:
>
> Shouldn't this and row 58 be secondary_vote?
>
> When I look at this page, I see the same vote for both primary and
> secondary - no matter how many times I restart the server or force refresh
> the browser.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/bethlakshmi/GBE2/pull/883/files/e6b509a77e38160589c4c8a4d96f41fe66bba9aa#r77934812>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AFCe4coJZp73A76jhCsxUhGaoFv6uIdiks5qn2kngaJpZM4J0xnd>
> .
>